CONTRIBUTING.md

```diff
@@ -45,28 +44,35 @@ There are a number of ways to get involved:
 
 In either case, we ask that you assign yourself to the relevant issue and open up [a draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) while you are working on your feature/bug-fix. This helps the Splink dev team keep track of developments and means we can start supporting you sooner!
 
-!!! note "Small PRs"
-    In the Splink dev team, we believe that [small Pull Requests](https://essenceofcode.com/2019/10/29/the-art-of-small-pull-requests/) make better code. They:
+You can always add further PRs to build extra functionality. Starting out with a minimum viable product and iterating makes for better software (in our opinion). It also helps get features out into the wild sooner.
 
-    - are more focused
-    - increase understanding and clarity
-    - are easier (and quicker) to review
-    - get feedback quicker
+To get set up for development locally, see the [development quickstart](https://moj-analytical-services.github.io/splink/dev_guides/dev_guides/changing_splink/development_quickstart.html).
 
-    You can always add further PRs to build extra functionality. Starting out with a minimum viable product and iterating  makes for better software (in our opinion). It also helps get features out into the wild sooner with regular Splink releases every other Wednesday.
+## Best practices
 
 When making code changes, we recommend:
-* [Recreating Splink's virtual environment](./docs/dev_guides/changing_splink/building_env_locally.md) to best replicate the conditions in which Splink will be used in practice. 
-* [Adding tests](./docs/dev_guides/changing_splink/testing.md) to ensure your code works as expected. These will be run through GitHub Actions when a PR is opened.
-* [Linting](./docs/dev_guides/changing_splink/lint_and_format.md) to ensure that code is styled consistently.
+
+- [Adding tests](https://moj-analytical-services.github.io/splink/dev_guides/changing_splink/testing.html) to ensure your code works as expected. These will be run through GitHub Actions when a PR is opened.
```

Comment:
These links were broken in the hosted version, though they were working in the GitHub version. I didn't see a way to make them relative and still work in both places.

Reply:
I think it's slightly nicer to link directly to the docs anyway. 

On this though (and something I'll make an issue on), we should check that the links in both our README and Contributing guide are valid. We have a [check](https://github.com/moj-analytical-services/splink/blob/master/.github/workflows/documentation.yml#L116C1-L120C56) in place for HTML files, which we should be able to extend to these markdown files.

Reply:
Ah, I think we are checking them, we just have a hacky solution in place.

That hacky solution is also why you were probably encountering issues - https://github.com/ADBond/splink/blob/179a3f67de3c1eaab1bea97214082146f46c6547/scripts/preprocess_markdown_includes.py#L48/

Reply:
Ah okay! I think with this change, I can remove the hack. Let me check this.

Reply:
I don't see the file you linked in the `master` branch 🤔